virtio-devices/src/balloon.rs

```diff
@@ -94,6 +95,42 @@ pub struct VirtioBalloonConfig {
     actual: u32,
 }
 
+#[derive(Copy, Clone, Debug)]
+struct PartPage {
+    addr: u64,
+    bitmap: u64,
```

Comment:
I don't think this is right.. With `u64`, you only got 64 bits, that is only able to track up to 64 pages. In another word, it would only page size that is equal or less than 256K (64*4K). You will need to use something like `Vec<u64>`.

Reply:
Yeah, It's not good here. A dynamic growth buff is better. I will change it.